### PR TITLE
made the sign-up method add first and last name to user

### LINKED ISSUE #77: Fix firebase Sign-up method
Change the sign-up method by allowing firebase to associate a first and last name to  the user's account upon sign-up.

### PR DESCRIPTION
**Pull request by YungiKim1999**
sign up method now allows users to sign up and associate a first and last name to their account

### CODE REVIEW

File: src/services/authService.js [hunk 1]
```diff
@@ -20,15 +20,27 @@ const signIn = async (email, password) => {
...
  * @param {*} email The email of the user as a string
  * @param {*} password The password of the user as a string
+ * @param {*} first_name The first name of the user as a string
+ * @param {*} last_name The last name of the user as a string
  */
-const signUp = async (email, password) => {
+const signUp = async (email, password, first_name, last_name) => {
     try{
-        await firebaseConnection
+        await firebaseApp
```
Comment (dongmeilim) [inline src/services/authService.js]: should probs leave this as firebaseConnection

File: src/services/authService.js [hunk 2]
```diff
@@ -20,15 +20,27 @@ const signIn = async (email, password) => {
...
         .auth()
-        .createUserWithEmailAndPassword(email, password);
+        .createUserWithEmailAndPassword(email, password).then(Credential => {
+            const db = firebaseApp.firestore();
```
Comment (dongmeilim) [inline src/services/authService.js]: this too

**Review by dongmeilim — CHANGES_REQUESTED** [review]
got some copy/paste errors, but looks good otherwise!!

File: src/services/authService.js (hunk 1 above)
Comment (YungiKim1999) [reply src/services/authService.js]: good catch~~

File: src/services/authService.js (hunk 2 above)
Comment (YungiKim1999) [reply src/services/authService.js]: on it

**Review by dongmeilim — APPROVED** [review]
nice job

**Review by max-gurr — APPROVED** [review]
Looks great!

**Pull request merged**